components/engine_rocks/Cargo.toml

```diff
@@ -56,6 +56,7 @@ txn_types = { path = "../txn_types", default-features = false }
 
 [dependencies.rocksdb]
 git = "https://github.com/tikv/rust-rocksdb.git"
+branch = "tikv-6.1"
```

Comment:
should we update it?

Reply:
Not sure, cc @tabokie

Reply:
In https://github.com/tikv/tikv/pull/12533